### PR TITLE
Add german translation of Sovereign Sight and Kitara cycle name

### PR DESCRIPTION
**Pull request by eljenso**

### CODE REVIEW

File: translations/de/pack/ss.de.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    },
+    {
+        "code": "21004",
+        "keywords": "Ice-Brecher - Killer",
+        "text": "1[credit]: Schalte eine Subroutine eines <strong>Wächters</strong> aus.\n2[credit]: +1 Stärke.\n[click]: Lege 1 Wirkungszähler auf Aufbläser oder entferne 1 Wirkungszähler von Aufbläser.\nStärke und Speicherkosten von Aufbläser werden für jeden Wirkungszähler auf dieser Karte um 1 erhöht.",
+        "title": "Aufbläser"
+    },
+    {
+        "code": "21005",
+        "flavor": "Lemi ist in den Slums von Jinja aufgewachsen und hat ein Vermögen mit illegalen Deals, gut abgepassten Wetten und geschickter Selbstdarstellung gemacht.",
```
Comment (gereons) [inline translations/de/pack/ss.de.json]: Lemi? Lewi vermutlich...

File: translations/de/pack/ss.de.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    },
+    {
+        "code": "21009",
+        "keywords": "Abteilung",
+        "text": "Wenn du zum ersten Mal in deinem Zug eine Karte installierst, darfst du eine zusätzliche Karte, die keine Agenda ist, vom HQ in jenes System oder jenes System schützend installieren (zahle dafür alle Kosten).",
+        "title": "Asa-Gruppe: Sicherheit durch Wachsamkeit"
+    },
+    {
+        "code": "21010",
+        "flavor": "\"Der durchschnittliche Einwohner von Nairobi spricht sechs Sprachen. Alle Kulturen, Sprachen und Volksgruppen von Ostafrika treffen hier aufeinander und verischen sich. Was im Leben nur langsam geschieht, kann im Labor beschleunigt werden. Wir bringen Friegen.\" - Dr. Muchina, Projektleiter",
```
Comment (gereons) [inline translations/de/pack/ss.de.json]: verischen -> vermischen

File: translations/de/pack/ss.de.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        "code": "21012",
+        "keywords": "Hinterhalt",
+        "text": "Genspleisser kann entwickelt werden.\nSobald der Runner auf Genspleisser zugreift, verursache 1 Netz-Schaden für jeden Entwicklungsmarker auf Genspleisser.\n<strong>[click], 3 Entwicklungsmarker:</strong> Füge Genspleisser als Agenda mit dem Wert 1 Agendapunkt deiner Punktezone hinzu.",
+        "title": "Genspleisser"
+    },
+    {
+        "code": "21013",
+        "flavor": "Dünn genug, um das Netz zu durchdringen. Tödlich genug, um deinen Geist zu durchdringen.",
+        "keywords": "Falle - Psi - AP",
+        "text": "[subroutine] Du und der Runner geben geheim 0[credit], 1[credit] or 2[credit]. Falls sich die Anzahl der von dir und vom Runner ausgegebenen Credits unterscheiden, verursache 2 Netz-Schaden; ansonsten verursache 1 Netz-Schaden. Zerstöre Mganga.",
```
Comment (gereons) [inline translations/de/pack/ss.de.json]: geben ... aus

File: translations/de/pack/ss.de.json [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    },
+    {
+        "code": "21014",
+        "flavor": "Manchmal muss man ein paar Unannehmlichkeiten in Kauf nehmen, wenn man ein Genom sequenzieren möchte.",
+        "text": "Zerstöre die obersten 2 Karten von F&E. Mische dann bis zu 4 Karten aus dem Archiv in F&E. Entferne Genotypisierung aus dem Spiel, statt sie zu zerstören.",
+        "title": "Genotypisierung"
+    },
+    {
+        "code": "21015",
+        "flavor": "\"Kapitalismus basiert auf zwei Dingen: unglaubliche Arroganz und Zensur des Widerspruchs. Damit kann ein Kon jede Umgebung erzeugen, die er will.\" - Freedom Khtemalo",
```
Comment (gereons) [inline translations/de/pack/ss.de.json]: Khtemalo -> Khumalo

**Comment by eljenso** [comment]
Thanks for review @gereons. Also found some minor typos in the English version, including "Khtemalo".

**Pull request merged**